clients/js/test/getAssetsByCreator.test.ts

```
@@ -23,4 +23,162 @@ DAS_API_ENDPOINTS.forEach((endpoint) => {
+
+    // And collection metadata should be present in the grouping for assets that have collections
+    const assetWithCollection = assets.items.find((asset) =>
+      asset.grouping?.some((group) => group.group_key === 'collection')
+    );
+    t.truthy(
+      assetWithCollection,
+      'Expected to find at least one asset with a collection'
+    );
+
+    // We've verified assetWithCollection exists with t.truthy above
+    const collectionGroup = assetWithCollection!.grouping.find(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)